packages/bootstrap-chat/src/FormResponse.jsx

```diff
@@ -115,6 +115,14 @@ class FormResponse extends React.Component {
             {question}
           </Col>
           <Col sm={5}>
+            onKeyPress={(e) => {
```

Comment:
This is 1) on the Col component now (it should be a prop on the FormControl component) and 2) it should be within the > tag of the component. Right now onKeyPress is after the ending > tag of the Col component